lib/lib_test.go

```diff
@@ -0,0 +1,282 @@
+package lib_test
+
+import (
+	"net/http"
+	"net/http/httptest"
+
+	"github.com/labstack/echo/engine/standard"
+	"github.com/spf13/viper"
+	"github.com/topfreegames/podium/api"
+	"github.com/topfreegames/podium/lib"
+	"github.com/topfreegames/podium/testing"
+	httpmock "gopkg.in/jarcoal/httpmock.v1"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var transport *http.Transport
+
+func GetDefaultTestApp() *api.App {
+	logger := testing.NewMockLogger()
+	app, err := api.GetApp("0.0.0.0", 8890, "../config/test.yaml", false, false, logger)
+	Expect(err).NotTo(HaveOccurred())
+	app.Configure()
+	return app
+}
+
+func initClient() {
+	if transport == nil {
+		transport = &http.Transport{DisableKeepAlives: true}
+	}
+}
+
+func InitializeTestServer(app *api.App) *httptest.Server {
+	initClient()
+	app.Engine.SetHandler(app.App)
+	return httptest.NewServer(app.Engine.(*standard.Server))
+}
+
+var _ = Describe("Lib", func() {
+
+	var a *api.App
```

Comment:
do we still need app? I understood that with httpmock we don't need podium server anymore.